src/VisualStudio/Core/Next/Remote/ServiceHubRemoteHostClient.cs

```diff
@@ -73,28 +73,16 @@ internal sealed partial class ServiceHubRemoteHostClient : RemoteHostClient
                 return;
             }
 
-            // don't block UI thread while initialize workspace host
-            var host = new WorkspaceHost(vsWorkspace, client);
-
-            // Initialize the remote side with whatever data we have currently for the workspace.
-            // As workspace changes happen, this host will get notified, and it can remote those
-            // changes appropriately over to the remote size.
-            await host.InitializeAsync().ConfigureAwait(false);
```

Comment:
Just checked.  InitializeAsync does three things:  

1. _client.TryCreateServiceSessionAsync 
2. IRemoteHostService.RegisterPrimarySolutionId
3. IRemoteHostService.UpdateSolutionIdStorageLocation

2 and 3 are very cheap (and shoudl probably just be merged into one call).

It's unclear ot me if '1' is expensive or not.  If it's cheap, then my change is totally safe to do.  If '1' is expensive then there's an issue here.  @heejaechang can you comment on this?